swdb2017/brain_observatory/get_pupil_size_data.py

```diff
@@ -1,25 +1,27 @@
 import pandas as pd
+import numpy as np
+from scipy.ndimage.filters import gaussian_filter
 
 def has_pupil_data_df(boc,targeted_structures = None,stimuli = None,cre_lines = None):
     '''Returns full dataframe with only experiments that have pupil data for desired targeted structures, stimuli classes, 
     cre lines.
     
     Parameters
     ----------
-            boc : BrainObservatory Cache from allensdk
-            //from allensdk.core.brain_observatory_cache import BrainObservatoryCache
-            //manifest_file = os.path.join(drive_path,'brain_observatory_manifest.json')
-            //boc = BrainObservatoryCache(manifest_file=manifest_file)
-            
-            targeted_structures : list of brain region acronyms 
-            
-            stimuli : list of stimulus class names
-            
-            cre_lines : list of cre-line names
+    boc : BrainObservatory Cache from allensdk
+    //from allensdk.core.brain_observatory_cache import BrainObservatoryCache
+    //manifest_file = os.path.join(drive_path,'brain_observatory_manifest.json')
+    //boc = BrainObservatoryCache(manifest_file=manifest_file)
+    
+    targeted_structures : list of brain region acronyms 
+    
+    stimuli : list of stimulus class names
+    
+    cre_lines : list of cre-line names
         
```

Comment:
the numpy format for docstrings is

variable: type
      description

you have

variable: type and description

this becomes useful when your types are actually defined elsewhere in your classes, and documentation tools can dynamically link to the documentation of those types, where when you mix the descriptions and the types its harder to make that happen automagically.